tests/PPC/TestLifting.cpp

```diff
@@ -241,6 +242,10 @@ int main(int argc, char **argv) {
   return RUN_ALL_TESTS();
 }
 
+
+inline const remill::DecodingContext kVLEContext =
```

Comment:
this we probably want to move to a header file like we have with Thumb
also provide another constant for non VLE context

Reply:
It looks like the only reason that Thumb does that is because it gets used in the decoding implementation and not just the tests. Since we're only using them in the tests, this seems fine right?